### PR TITLE
Update multi thumb slider

### PR DESCRIPTION
**Pull request by jongund**
Updated multi-thumb slider example:

1. Using SVG for the slider controls to support high contrast
2. Added accessibility feature section
3. Additional mobile version using input elements to set range
4. Updated code use the APG coding practices

[Preview multi-thumb slider](https://raw.githack.com/w3c/aria-practices/update-multi-thumb-slider/examples/slider/slider-multithumb.html)
[Preview multi-thumb slider with mobile support](https://raw.githack.com/w3c/aria-practices/update-multi-thumb-slider/examples/slider/slider-multithumb-mobile.html)

### CODE REVIEW

**Comment by nschonni** [comment]
`npm run fix` should correct all the minor spacing issues

**Comment by jongund** [comment]
@nschonni 
Thanks for the help, but that command changes a lot more JS files than I want changed in this pull request.

**Comment by nschonni** [comment]
If you're on Windows like me, it does change a whole bunch of files when it "fixes" the CRLF, but when you stage the changes, git figures out that there are now changes and looks less scary

**Comment by jongund** [comment]
@nschonni @spectranaut 
The regression tests run fine on my local machine, not sure why they fail on travis.

**Comment by nschonni** [comment]
Checking out the branches, the tests worked for me locally too, but when I tried to rebase I ran into a bunch of conflicts. I think it's likely failing because their are issues when it's testing on the files as a merge into master

**Comment by spectranaut** [comment]
> Checking out the branch, the tests worked for me locally too, but when I tried to rebase I ran into a bunch of conflicts. I think it's likely failing because their are issues when it's testing on the files as a merge into master

I've seen this happen to me before as well.. maybe merge master in and push?

File: examples/slider/js/slider-multithumb.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+  }
+
+  onInputChange(event) {
+    var tgt = event.currentTarget,
+      value = tgt.value,
+      isNumber = typeof parseInt(value) === 'number';
+
+    //    console.log('\n[onInputChange][   label]: ' + tgt.getAttribute('aria-label'));
+    //    console.log('[onInputChange][isNumber]: ' + isNumber);
+    //    console.log('[onInputChange][   value]: ' + value + ' ' + value.length + ' ' + typeof value);
```
Comment (spectranaut) [inline examples/slider/js/slider-multithumb.js:312]: Might want to get ride of these :)

**Review by spectranaut — APPROVED** [review]
The code and tests look good to me, we should just try to see if we can get those test passing after the conflict is resolved!

File: examples/slider/js/slider-multithumb.js (hunk 1 above)
Comment (jongund) [reply examples/slider/js/slider-multithumb.js]: @spectranaut 
Thanks for pointing this out, I just pushed an update that removed those lines

**Comment by mcking65** [comment]
@jongund 
Could you please put the new example into a separate PR?

**Pull request closed without merging**